.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
No new line EOF

Reply:
Never again will you have to see another gross line ending.